dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -62,11 +62,12 @@ derby/10.16.1.1//derby-10.16.1.1.jar
 derbyshared/10.16.1.1//derbyshared-10.16.1.1.jar
 derbytools/10.16.1.1//derbytools-10.16.1.1.jar
 dropwizard-metrics-hadoop-metrics2-reporter/0.1.2//dropwizard-metrics-hadoop-metrics2-reporter-0.1.2.jar
+error_prone_annotations/2.18.0//error_prone_annotations-2.18.0.jar
```

Comment:
Do we have to add this dependency?

Reply:
short answer, no, but Guava changed it to compile scope in https://github.com/google/gson/pull/2346

I'm okay with either include or exclude it from Spark classpath

Reply:
+1 for removal